autotools error with non-standard libtpms install location

With v0.4.0-24-g4827574

```
PKG_CONFIG_PATH=$INST_PATH/lib/pkgconfig ./autogen.sh --prefix=$INST_PATH
```

```
checking for LIBTPMS... yes
checking for TPMLIB_ChooseTPMVersion in -ltpms... no
configure: error: "libtpms 0.6 or later is required"
```

```
ls $INST_PATH/lib/pkgconfig/
libtpms.pc
```

```
autoreconf -V
autoreconf (GNU Autoconf) 2.69
```

--- a/configure.ac
+++ b/configure.ac
@@ -168,7 +168,7 @@ PKG_CHECK_MODULES(
 	,
 	AC_MSG_ERROR("no libtpms.pc found; please set PKG_CONFIG_PATH to the directory where libtpms.pc is located")
 )
-LDFLAGS="$LDFLAGS $LIBTPMS_LDFLAGS"
+LDFLAGS="$LDFLAGS $LIBTPMS_LIBS"
 CFLAGS="$CFLAGS $LIBTPMS_CFLAGS"
 AC_CHECK_LIB(tpms,
              TPMLIB_ChooseTPMVersion,[true],

